client/client.go

```diff
@@ -74,6 +76,9 @@ type WrapperV2 struct {
 
 // NewV2 creates a client based on the latest gsclientgen version.
 func NewV2(conf *Configuration) (*WrapperV2, error) {
+	if conf.AuthHeaderGetter == nil {
+		conf.AuthHeaderGetter = func() (string, error) { return "", nil }
+	}
```

Comment:
In what situations will returning an empty string provide useful? Is that for unauthenticaded access of something?

Reply:
... probably for the `CreateAuthToken` function?

Reply:
This is really just in case someone does not provide a Getter during initialization of this client. (The tests for example do this)

I could throw an error somehow and make it a required configuration parameter. But at least in this case we don't get a panic when it tries to run a non existing function.

I could instead throw an error, but this error would only show up when gsctl is run, and not at compile time.

Reply:
> .. probably for the CreateAuthToken function?

Nope, the `CreateAuthToken` function does not use `SetParamsWithAuthorization` so it wont end up calling the `AuthHeaderGetter` either.